packages/flutter/test/material/search_test.dart

```diff
@@ -40,6 +40,29 @@ void main() {
     TestDefaultBinaryMessengerBinding.instance!.defaultBinaryMessenger.setMockMethodCallHandler(SystemChannels.platform, null);
   });
 
+  testWidgets('Changing query moves cursor to the end of query',
+      (WidgetTester tester) async {
```

Comment:
Nit: for consistency with the other tests in the file, this should all be on a single line:

```
  testWidgets('Changing query moves cursor to the end of query', (WidgetTester tester) async {
```